docs/reference/training/experiment-config-reference.rst

```diff
@@ -280,6 +280,48 @@ trials will not be restarted and will be marked as errored. An experiment is con
 if at least one of its trials completes without errors. The default value for ``max_restarts`` is
 ``5``.
 
+.. _config-log-policies:
+
+``log_policies``
+================
+
+Optional. The ``log_policies`` parameter is a list of policies that allows certain actions to be
+taken when a trial reports a log that matches the pattern. The pattern is a regex used by the Go
+language. See `this reference <https://github.com/google/re2/wiki/Syntax>`__ for details on the
+regex. The pattern regex is matched line by line against logs reported by the trial. The following
+actions can be taken
```

Comment:
``log_policies``
================

Optional. Defines actions in response to trial logs matching specified regex patterns (Go language syntax). For more information about the syntax, you can visit this `RE2 reference page <https://github.com/google/re2/wiki/Syntax>`__. Actions include: